src/checkPassword.test.js

```diff
@@ -8,12 +8,37 @@ describe(`Function 'checkPassword':`, () => {
   });
 
   it(`should return boolean`, () => {
+    const result = checkPassword('Password1!');
 
+    expect(typeof result).toBe('boolean');
   });
 
   it(`should return 'true' for the valid password with 8 characters`, () => {
+    expect(checkPassword('Password1!')).toBeTruthy();
+  });
+
+  it(`should return 'false' for the invalid password
+  with cyrrilic chars`, () => {
+    expect(checkPassword('Пасворд1!')).toBeFalsy();
+  });
+
+  it(`should return 'false' for 
+  the invalid password with less then 8 characters`, () => {
+    expect(checkPassword('Qwert!1')).toBeFalsy();
+  });
```

Comment:
Don't forget about boundary values! 8-16, so your tests should include verification for 7, 8, 16, 17 chars including all required symbols, fix it